raven/src/main/java/org/opencadc/raven/RavenInitAction.java

```diff
@@ -91,6 +103,7 @@
     private static final String RAVEN_KEY = "org.opencadc.raven";
 
     static final String JNDI_DATASOURCE = "jdbc/inventory"; // context.xml
+    static final String JNDI_AVAILABILITY_KEY = RAVEN_KEY + ".availabilities";
```

Comment:
RAVEN_KEY is for config items and not suitable for a JNDI key. The cadc-rest lib supplies a string that can be used as a prefix for JNDI keys (componentID).

Reply:
Used the appName from the InitAction & RestAction to generate the JNDI key.